Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.21 AS builder
+
+WORKDIR /building
+COPY . .
+
+ARG APP
+RUN make ${APP}
+
+FROM alpine:3.18 AS runtime
+
+ARG APP
+RUN addgroup -g 1000 -S ${APP} && adduser -u 1000 -S ${APP} -G ${APP} --home /app \
+ && echo -e "#!/bin/sh\nexec /usr/local/bin/${APP} \$@" > /app/entrypoint.sh \
+ && chmod +x /app/entrypoint.sh
+
+FROM alpine:3.18
+
+ARG APP
+ARG TITLE
+LABEL org.opencontainers.image.authors="fatedier <fatedier@gmail.com>"
```

Comment:
Why do we need these labels? What benefits will it bring?

Reply:
Best practice, a docker image tag is a also only a "label", those labels make it easier for your users what's inside this image.
The specific app version would be good too, but a bit harder to integrate in this case.
https://snyk.io/blog/how-and-when-to-use-docker-labels-oci-container-annotations/

Reply:
I do not prefer to add these redundant information.